controllers/admin/AdminSearchController.php

```diff
@@ -67,8 +67,17 @@ public function postProcess()
         $action = Tools::getValue('action');
         if ($action == 'redirectToProduct') {
             $id_product = (int) Tools::getValue('id_product');
-            $link = $this->context->link->getAdminLink('AdminProducts', false, ['id_product' => $id_product]);
-            Tools::redirectAdmin($link);
+            $product = new Product($id_product);
+            if (Validate::isLoadedObject($product) && Tools::getIsset('statusproduct')) {
+                if ($product->toggleStatus()) {
+                    $this->confirmations[] = $this->trans('The status has been updated successfully', [], 'Admin.Notifications.Success');
```

Comment:
```suggestion
                    $this->confirmations[] = $this->trans('The status has been updated successfully.', [], 'Admin.Notifications.Success');
```

Reply:
Hi @PululuK, I would recommend the version with a dot at the end so that both notifications (error and success) stay consistent. :-)

Reply:
Thank you @LouiseBonnard  !